hello/RelatedArtistsPlaylist.py

```diff
@@ -23,20 +23,167 @@
 user = spotifyObject.current_user()             # get current app user
 
 
+randsongs = []  # empty list to store songs that will go into playlist
+songs = []      # empty list to store songs from related artists
+
 class relatedArtistsPlaylistClass():
 
     def unclepostyPlaylist(self):
+        searchResults = spotifyObject.search('Post+Malone', 1, 0, 'artist')     # search for selected artist object
+        getArtist = searchResults['artists']['items'][0]
+        artistId = getArtist['id']          # get slected artist's Id
+
+        relatedArtists = spotifyObject.artist_related_artists(artistId)
+        artists = relatedArtists['artists']     # search for related artists
+
+        for x in artists:
+            artistName = (x['name'])
+            artistSearch = spotifyObject.search(artistName, 2, 0 ,'track')      # search all related artists
+            tracks = artistSearch['tracks']['items']
+            for x in tracks:
+                songs.append(x['uri'])                                          # get songs by realted artists
+                # print(songs)
+    
+
+
+    def selenagomezPlaylist(self):
+        searchResults = spotifyObject.search('Selena+Gomez', 1, 0, 'artist')     # search for selected artist object
+        getArtist = searchResults['artists']['items'][0]
+        artistId = getArtist['id']          # get slected artist's Id
+
+        relatedArtists = spotifyObject.artist_related_artists(artistId)
+        artists = relatedArtists['artists']     # search for related artists
```

Comment:
These playlists will always create the same playlist because songs aren't selected randomly.